File: python/1924.py
months = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
days = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"]

x, y = map(int, input().split())

sum = 0
for i in range(x - 1):
    sum += months[i]
sum += y-1

print(days[int(sum % 7)])

File: cpp/algorithm/algorithm/1924.cpp
#include <iostream>

using namespace std;

int months[] = {31,28,31,30,31,30,31,31,30,31,30,31};
string day[] = {"MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"};

int main(){
    cin.tie(0);
    cout.tie(0);

    int x, y;
    cin >> x >> y;

    int sum = 0;

    for(int i = 0; i < x-1; i++){
        sum += months[i];
    }
    sum += y-1;

    cout << day[sum % 7] << '\n';

    return 0;
}